statescript/executor.go

```diff
@@ -79,7 +79,7 @@ func (l Launcher) getTimeout() time.Duration {
 	if l.Timeout != 0 {
 		return time.Duration(l.Timeout) * time.Second
 	}
-	log.Debugf("statescript: timeout for executing scripts is not defined; using default of %s seconds", defaultStateScriptTimeout)
+	log.Debugf("Statescript: The timeout for executing scripts is not defined; using default of %s seconds", defaultStateScriptTimeout)
```

Comment:
I think these are weird. I would suggest using `statescript` (as it was), `StateScript` or `State script`.